database/questions.sql

```diff
@@ -0,0 +1,15 @@
+CREATE DATABASE questions_database;
```

Comment:
Nice schema file, we may want to add seeding to this file for when the server needs to restart